python/kserve/kserve/model_server.py

```diff
@@ -114,11 +117,18 @@ def __init__(self, http_port: int = args.http_port,
         # Logs can be passed as a path to a file or a dictConfig.
         # We rely on Uvicorn to configure the loggers for us.
         if configure_logging:
-            self.log_config = log_config if log_config is not None else KSERVE_LOG_CONFIG
+            # log_config could be a dictionary serialized as string,
```

Comment:
I'd suggest if we want to enable the user to provide a full logging config without needing to create a log config file we should do it via an environment variable. I don't think it's a great practice to pass really large strings as command line arguments. Maybe we can look for a `KSERVE_LOG_CONFIG` environment variable and expect that to be a JSON formatted string containing the dict config.